[Card Games]: Exercise Improvements (#2820)
exercism/python: exercises/concept/card-games/.docs/instructions.md

## 5. Alternate Averages

In Black Joe, speed is important. Elyse is going to try and find a faster way of finding the average.

She has thought of two ways of getting an _average-like_ number:

- Take the average of the _first_ and _last_ number in the hand.
- Using the median (middle card) of the hand.
  

> Implement the function `approx_average_is_average(<hand>)`, given `hand`, a list containing the values of the cards in your hand.

Reviewer: ```suggestion
Implement the function `approx_average_is_average(<hand>)` that  takes as a parameter a `list` containing the values of the cards in the "hand".
```

> Return `True` if either _one_ `or` _both_ of the, above named, strategies result in a number _equal_ to the _actual average_.

Reviewer: ```suggestion
Return `True` if either **_one  or _both__** of the average-like strategies noted above result in a number _equal_ to the _actual average_.
```
